ui/src/app/components/access-renewal-notification.spec.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+
+import {ProfileStubVariables} from 'testing/stubs/profile-api-stub';
+import {Profile, RenewableAccessModuleStatus} from 'generated/fetch';
+import {
+    maybeDaysRemaining,
+    MILLIS_PER_DAY,
+    NOTIFICATION_THRESHOLD_DAYS
+} from "./access-renewal-notification";
+import ModuleNameEnum = RenewableAccessModuleStatus.ModuleNameEnum;
+
+// 10 minutes, in millis
+const SHORT_TIME_BUFFER = 10 * 60 * 1000;
+
+// return a time (in epoch millis) which is today + `days` days, plus a short buffer
+const todayPlusDays = (days: number): number => {
+    return Date.now() + (MILLIS_PER_DAY * days) + SHORT_TIME_BUFFER ;
```

Comment:
Nit: I'm surprised by four spaces here, though yarn lint doesn't complain.

Reply:
yeah I'm not sure the linter is actually running on the test files